prerequisites/install-functions/build_opencoarrays.sh

```diff
@@ -21,7 +21,7 @@ build_opencoarrays()
   # Set FC to the MPI implementation's gfortran command with any preceding path but without any subsequent arguments:
   FC="${MPIFC_show%%gfortran*}"gfortran
   # Set CC to the MPI implementation's gcc command...
-  CC="${MPICC_show%%gcc*}"gcc
+  CC="${MPICC_show%%gcc *}"gcc
```

Comment:
> The wild card in the build script would eat the path component
of the installed gcc 6.1.0 build and set the var to a directory
instead of .../bin/gcc

But after eating everything, it adds back gcc.... on my system if you add the space then CC just gets set to the entire `mpicc -show` so I don't think this is actually a viable fix... I'll have to look at #306 and dig around a bit to understand why this is causing issues for you.